src/platform_linux/shmem.c

```diff
@@ -104,20 +128,10 @@ typedef struct shmem_heap {
    platform_spinlock shm_mem_frags_lock;
    // Protected by shm_mem_frags_lock. Must hold to read or modify.
    shm_large_frag_info shm_large_frags[SHM_NUM_LARGE_FRAGS];
-   uint32              shm_nlarge_frags_tracked;
-
-   uint32 shm_nlarge_frags_inuse;
-   uint32 shm_nlarge_frags_inuse_HWM;
-   size_t shm_used_by_large_frags; // Actually reserved
-   size_t shm_nfrees;              // # of calls to free memory
-   size_t shm_nfrees_last_frag;    // Freed last small-fragment
 
-   size_t shm_total_bytes; // Total size of shared segment allocated initially.
-   size_t shm_free_bytes;  // Free bytes of memory left (that can be allocated)
-   size_t shm_used_bytes;  // Used bytes of memory left (that were allocated)
-   size_t shm_used_bytes_HWM; // High-water mark of memory used bytes
```

Comment:
This set of deleted stats-fields are now dropped into common `shminfo_usage_stats{}` struct; listed below.